stan/math/prim/scal/fun/distance.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_DISTANCE_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_DISTANCE_HPP
+
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/fun/abs.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Returns the squared distance.
+     *
+     * @param x1 First vector.
+     * @param x2 Second vector.
+     * @return Distance between two scalars
+     * @throw std::domain_error If the vectors are not the same
```

Comment:
Is this a cut-and-paste error?  It looks like it checks for finiteness.

The arguments say vector, but the return says scalars.  

I hope the definition is only intended for scalars because otherwise it won't work.

Reply:
Yes, this was copy-pasted from stan/math/prim/scal/fun/squared_distance. I assume there the description is erroneous as well then.